### ISSUE TITLE
Test DC6

### ISSUE THREAD

**Issue by celskeggs** [sipb/homeworld#128]
This is about wiping and re-deploying the cluster.

Things to make sure to test:
 - [ ] Flannel still works after #94 (upgrade to 0.9.0)
 - [ ] etcd still works after #93 (upgrade to 3.2.9)

This issue is just here as a reminder for the DC6 testing cycle.

**Comment by celskeggs** [comment]
The cluster is getting redeployed frequently enough that there's no point in having this as a separate issue.
